src/metabase/notification/payload/core.clj

```diff
@@ -20,43 +22,115 @@
     [:updated_at   {:optional true} :any]]
    [:multi {:dispatch :payload_type}
     ;; system event is a bit special in that part of the payload comes from the event itself
-    [:notification/system-event [:map
-                                 [:payload
-                                  [:map {:closed true}
-                                   ;; TODO: event-info schema for each event type
-                                   [:event_topic [:fn #(= "event" (-> % keyword namespace))]]
-                                   [:event_info  [:maybe :map]]]]]]
-
+    [:notification/system-event
+     [:map
+      [:payload
+       [:map {:closed true}
+        ;; TODO: event-info schema for each event type
+        [:event_topic [:fn #(= "event" (-> % keyword namespace))]]
+        [:event_info  [:maybe :map]]]]]]
+    [:notification/alert
+     [:map
+      ;; replacement of pulse
+      [:alert      [:map
```

Comment:
this `alert` and `dashboard_subscription` map will have their own tables when we migrate the schema. See [this](https://www.notion.so/metabase/Notification-high-level-design-9620a4a69db0494f806a146cc9174964?pvs=4#618b5f4b2228459eb3531fb0ada02738) high level schema